src/components/data-entry/DataSetStateButton.tsx

```diff
@@ -78,6 +86,7 @@ const DataSetStateButton: React.FunctionComponent<DataSetStateButtonProps> = pro
             {!dataSetInfo.isOpen && userCanReopen && (
                 <>
                     <Button
+                        style={{ marginRight: 20 }}
```

Comment:
I've changed className for style, it seems MUI overrides marings when using the class.